### PR TITLE
Add Info Security to Project Review and Edit 

### PR DESCRIPTION
**Pull request by wloforteBroad**
On behalf of @tadeork 

**Addresses:**
(https://broadinstitute.atlassian.net/browse/BTRX-586)

The Security questions already added to the creation flow are now on the Review Page and are editable.

---

- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Get a thumb from @rushtong
- [ ] **Submitter**: Merge to develop using github's "Squash and Merge" feature
- [ ] **Submitter**: Delete branch after merge

### CODE REVIEW

File: src/main/webapp/components/SecurityReview.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        accessible: true,
+        textCompliance: true,
+        textSensitive: true,
+        textAccessible: true,
+      },
+    };
+    this.formHasError = this.formHasError.bind(this);
+  }
+
+  componentDidMount() { }
```
Comment (rushtong) [inline src/main/webapp/components/SecurityReview.js]: Do you need this?

File: src/main/webapp/components/SecurityReview.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+        edit: this.props.edit,
+        review: this.props.review
+      })
+    ])
+  };
+
+  render() {
+    if (this.state.hasError) {
+      // You can render any custom fallback UI
+      return h1({}, ["Something went wrong."]);
```
Comment (rushtong) [inline src/main/webapp/components/SecurityReview.js]: This seems pretty unhelpful. Can we add something explanatory here?

File: src/main/webapp/consentGroup/NewConsentGroup.js [hunk 3]
```diff
@@ -376,6 +376,7 @@ class NewConsentGroup extends Component {
   };
 
   handleInfoSecurityValidity(isValid) {
+    console.log('handleInfoSecurityValidity', isValid);
```
Comment (rushtong) [inline src/main/webapp/consentGroup/NewConsentGroup.js]: rm

File: src/main/webapp/components/SecurityReview.js (hunk 2 above)
Comment (tadeork) [reply src/main/webapp/components/SecurityReview.js]: I agree. This should be more explanatory. But we're using this as a simple way to display a error when the component can't load. Probably we need to fix this later to a better explanation, in all the components this messages is present. At least for us this message it's helpful for now.

**Pull request merged**